extension-androidauto/src/main/java/com/mapbox/maps/extension/androidauto/CarMapSurfaceOwner.kt

```diff
@@ -121,22 +140,40 @@ internal class CarMapSurfaceOwner(
     }
   }
 
+  /**
+   * @see SurfaceCallback.onStableAreaChanged
```

Comment:
Support for this is added here https://github.com/mapbox/mapbox-maps-android/pull/1648